squish/src/lib.rs

```diff
@@ -333,4 +333,60 @@ mod tests {
         let estimate = Format::Bc3.compressed_size(15, 30);
         assert_eq!(estimate, 512);
     }
+
+    // A colour test-pattern (RGB) with the first row in one colour,
+    // the second in another and the third and last row in a third colour.
+    static DECODED_BLOCK_COLOUR_4X4: &[u8] = &[
+        255, 150, 74, 255, 150, 74, 255, 150, 74, 255, 150, 74, // row 0
+        255, 120, 52, 255, 120, 52, 255, 120, 52, 255, 120, 52, // row 1
+        255, 105, 41, 255, 105, 41, 255, 105, 41, 255, 105, 41, // row 2
+        255, 105, 41, 255, 105, 41, 255, 105, 41, 255, 105, 41, // row 3
+    ];
+
+    // BC1 data created with AMD Compressonator v4.1.5083 and is the same as libsquish
+    static ENCODED_BLOCK_COLOUR_4X4: [u8; 8] = [0xA9, 0xFC, 0x45, 0xFB, 0x00, 0xFF, 0x55, 0x55];
```

Comment:
I'm not a huge fan of having data for all tests in the same namespace like this. Also the names for the variables don't really explain what is tested with them.